pkg/controller/config/config.go

```diff
@@ -93,12 +108,37 @@ func (c *XdsConfig) Init() error {
 
 	log.Infof("service node %v connect to discovery address %v", c.ServiceNode, c.DiscoveryAddress)
 
+	c.Metadata.Namespace = podNamespace
+	c.Metadata.ClusterID = cluster.ID(clusterID)
+	c.Metadata.InstanceIPs = []string{ip}
+	// TODO: add labels to support localiy load balancing
+	c.Metadata.Labels = nil
+	c.Metadata.MeshID = "cluster.local"
```

Comment:
Should `MeshID` also be allowed to get from env or command line flag?

Reply:
sgtm